### PR TITLE
Release updates for 3.2

### PR DESCRIPTION
**Pull request by eteq**
Note that #328 already updated the team

### CODE REVIEW

File: announcements/release-3.2.html [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+		&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://github.com/astropy/astropy/issues">https://github.com/astropy/astropy/issues</a>
+		</p>
+		<p>
+		Over 300 developers have contributed code to Astropy so far, and you can find out more about the team behind Astropy here:
+		</p>
+		<p>
+		&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://www.astropy.org/team.html">http://www.astropy.org/team.html</a>
+		</p>
+		<p>
+			As a reminder, Astropy v2.0 (our long term support release) will continue to be supported with bug fixes (but no new features) until the end of 2019, so if you need to use Astropy in a very stable environment, you may want to consider staying on the v2.0.x set of releases (for which we have recently released v2.0.10).
```
Comment (bsipocz) [inline announcements/release-3.2.html]:
```suggestion
			As a reminder, Astropy v2.0 (our long term support release) will continue to be supported with bug fixes (but no new features) until the end of 2019, so if you need to use Astropy in a very stable environment, you may want to consider staying on the v2.0.x set of releases (for which we have recently released v2.0.13).
```

File: announcements/release-3.2.html [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+		It is part of the Astropy Project, which aims to foster an ecosystem of
+		interoperable astronomy packages for Python.
+		</p>
+		<p>
+		New and improved major functionality in this release includes:
+		<ul>
+			<li>New Sub-package for Time Series</li>
+			<li>New SI/CODATA 2018 Constants</li>
+			<li>Additions and changes to Ecliptic Transformations</li>
+			<li>Table performance improvements and change in meta handling</li>
```
Comment (astrofrog) [inline announcements/release-3.2.html]: meta -> metadata (minor)

File: announcements/release-3.2.html [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+		</p>
+		<p>
+		If you use Astropy directly for your work, or as a dependency to another
+		package, please remember to acknowledgment it by citing the appropriate
+		Astropy paper.  For the most up-to-date suggestions, see 
+		<a href="../acknowledging.html">the acknowledgement page</a>, but as of
+		this release the recommendation is:
+		</p>
+		<p class="citation"><cite>This research made use of Astropy, a
+		community-developed core Python package for Astronomy (<a href="https://doi.org/10.3847/1538-3881/aabc4f" target="_blank">Astropy Collaboration,
```
Comment (astrofrog) [inline announcements/release-3.2.html]: On the main website the recommended acknowledgment includes both papers (https://www.astropy.org/acknowledging.html)

File: announcements/release-3.2.html (hunk 2 above)
Comment (eteq) [reply announcements/release-3.2.html]: Already in the what's new, but c'est la vie, will fix it here.

File: announcements/release-3.2.html (hunk 3 above)
Comment (eteq) [reply announcements/release-3.2.html]: Hmm. I've always viewed that as the "generic" blurb (i.e., if you're not sure), but if you know it's a specific version (which is what this announcement refers to) you pick the most recent one...

Comment (astrofrog) [reply announcements/release-3.2.html]: Ok no problem!

**Pull request merged**